gen_schema.py

```diff
@@ -0,0 +1,41 @@
+from typing import Annotated, Any, Sequence
+
+from pydantic import Field, schema, schema_json_of
+from pydantic.fields import ModelField
+from pydantic.schema import SkipField
+
+from kpops.components.base_components.kafka_connect import (
+    KafkaSinkConnector,
+    KafkaSourceConnector,
+)
+from kpops.components.streams_bootstrap.producer.producer_app import ProducerApp
+from kpops.components.streams_bootstrap.streams.streams_app import StreamsApp
+
+original_field_schema = schema.field_schema
```

Comment:
Move to the `field_schema` method?

Reply:
that doesn't work. we're monkeypatching one of pydantic's builtin functions and before that we have to store the original function somewhere. That's what this line does